### PR TITLE
Domain fixes, updated wording, bug fixes

### PR DESCRIPTION
**Pull request by brittnilorton**
This includes some domain issues, wording updates, fixes right sided limit, and a few other minor bug fixes

### CODE REVIEW

File: Contrib/CCCS/PreCalculus/2.5/CCD_CCCS_Openstax_AlgTrig_AT-1-001-AS_2_5_32.pg [hunk 1]
```diff
@@ -45,6 +45,8 @@ $f=Formula("$a*x^2+$b*x+$c")->reduce;
 
 $ans = ($b)**2-4*$a*$c;
 
+$ans=0;
+
```
Comment (drdrew42) [inline Contrib/CCCS/PreCalculus/2.5/CCD_CCCS_Openstax_AlgTrig_AT-1-001-AS_2_5_32.pg:49]:
```suggestion
```
I think you do not want to overwrite the value of $ans here.

**Review by drdrew42 — CHANGES_REQUESTED** [review]
Thanks for the updates! I found a couple of places where I think the answer is not being correctly set, and one minor typo...

File: Contrib/CCCS/CalculusOne/03.9/CCD_CCCS_Openstax_Calc1_C1-2016-002_3_9_350.pg [hunk 2]
```diff
@@ -33,23 +33,19 @@ $showPartialCorrectAnswers = 1;
...
-
+ 
 $ans=Formula("(x^$a-1)^ln(x)(ln(x^$a-1)/x+ln(x)($a*x^($a-1)/(x^$a-1)))")->reduce;
 
 ###########################
 #  Main text
-## I hope you get [`[$ans1]`], [`[$ans2]`], and [`[$ans3]`].
 BEGIN_PGML
 
 Use logrithmic differentiation to find `\frac{dy}{dx}` for [``y=[$f]``].
```
[the remark this answers is not in the document]
Comment (brittnilorton) [reply Contrib/CCCS/CalculusOne/03.9/CCD_CCCS_Openstax_Calc1_C1-2016-002_3_9_350.pg:51]: This is updated - thanks for noticing!

File: Contrib/CCCS/PreCalculus/2.5/CCD_CCCS_Openstax_AlgTrig_AT-1-001-AS_2_5_32.pg (hunk 1 above)
Comment (brittnilorton) [reply Contrib/CCCS/PreCalculus/2.5/CCD_CCCS_Openstax_AlgTrig_AT-1-001-AS_2_5_32.pg:49]: You are right, did not mean to do that, it is fixed now.

File: Contrib/CCCS/CalculusOne/04.8/CCD_CCCS_Openstax_Calc1_C1-2016-002_4_8_392.pg [hunk 3]
```diff
@@ -41,11 +41,11 @@ $ans =0;
 #  Main text
 
 BEGIN_PGML
-Evaluate the limit, [``\lim_{x\to 0}x^{\left(\dfrac{[$a]}{\cos(x)}\right)}``], either with l'Hopital's rule or previously learned methods.
+Evaluate the limit, [``\lim_{x\to 0^+}x^{\left(\dfrac{[$a]}{\cos(x)}\right)}``], either with l'Hopital's rule or previously learned methods.
 
 
 
-[``\lim_{x\to 0}x^{\left(\dfrac{[$a]}{\cos(x)}\right)}=``][_______________]{$ans } (Enter a number,  `\infty,-\infty,`  or "DNE".) [@ AnswerFormatHelp("numbers") @]*
+[``\lim_{x\to 0^+}x^{\left(\dfrac{[$a]}{\cos(x)}\right)}=``][_______________]{$ans } (Enter a number,  `\infty,-\infty,`  or "DNE".) [@ AnswerFormatHelp("numbers") @]*
```
[the remark this answers is not in the document]
Comment (brittnilorton) [reply Contrib/CCCS/CalculusOne/04.8/CCD_CCCS_Openstax_Calc1_C1-2016-002_4_8_392.pg:48]: Um, I don't think so. This is x ^(a/cos(x)) not sin(x)...and so no LHR is required as this is not an indeterminate form and so the limit is just zero....maybe I am wrong?

[the remark this answers is not in the document]
Comment (drdrew42) [reply Contrib/CCCS/CalculusOne/04.8/CCD_CCCS_Openstax_Calc1_C1-2016-002_4_8_392.pg:48]: I *knew* I was missing something here... I totally overlooked the "x^"

**Review by drdrew42 — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**